plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -986,57 +985,58 @@ export default Component.extend({
     }
 
     this.set("_nextStagedMessageId", this._nextStagedMessageId + 1);
-    const cooked = this.cook(message);
-    const stagedId = this._nextStagedMessageId;
-    let data = {
-      message,
-      cooked,
-      staged_id: stagedId,
-      upload_ids: uploads.map((upload) => upload.id),
-    };
-    if (this.replyToMsg) {
-      data.in_reply_to_id = this.replyToMsg.id;
-    }
+    return this.chat.loadCookFunction(this.site.categories).then((cook) => {
```

Comment:
I hope we can move this out of live pane at some point too, surely we can just do this once. Noticed this when I was digging in yesterday

Reply:
Yes totally agreed